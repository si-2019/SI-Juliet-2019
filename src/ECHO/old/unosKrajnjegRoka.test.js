import React from 'react';
import { shallow } from 'enzyme';
import UnosKrajnjegRoka from "./unosKrajnjegRoka.js";
import "../setupTests.js";


it('Provjerava da li postoji naslov.', () => {
    const wrapper = shallow(<UnosKrajnjegRoka />);
    const tekst = <label>Unesite datum i vrijeme za krajnji rok unosa i promjene slobodnih termina za nastavno osoblje:</label>;
    // expect(wrapper.contains(welcome)).toBe(true);
    expect(wrapper.contains(tekst)).toEqual(true);
  });

 
  it("Da li se prikazuju sve komponente", () => {
    const component = shallow(<UnosKrajnjegRoka />);
    expect(component).toMatchSnapshot();
  });

  it("Da li ima postoji kalendar za unos roka?", () => {
    const wrapper = shallow(<UnosKrajnjegRoka />);
    expect(wrapper.find("#KalendarZaUnos").exists()).toBe(true);
  });