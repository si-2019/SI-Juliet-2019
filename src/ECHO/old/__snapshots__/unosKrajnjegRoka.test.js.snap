// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Da li se prikazuju sve komponente 1`] = `ShallowWrapper {}`;
